samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -501,6 +523,26 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
+.. _nrf_cloud_mqtt_multi_service_building_wifi:
+
+Building with nRF7002 EK Wi-Fi scanning support (for nRF9160 DK)
+================================================================
+
+To build the sample with nRF7002 EK Wi-Fi scanning support, use the ``-DSHIELD=nrf7002_ek``, ``-DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay`` and  ``-DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf`` options.
+
+This will automatically enable the Wi-Fi location tracking method.
```

Comment:
```suggestion
This enables the Wi-Fi location tracking method.
```

Reply:
I think it is important to mention that this is automatic, since my goal here is to communicate that the user doesn't need to set this KConfig setting themselves, whereas they otherwise usually would. Though maybe there is another way to phrase it?

Reply:
Fine to mention this happens automatically -> This enables the Wi-Fi location tracking method automatically.